yarn.lock

```diff
@@ -2451,15 +2451,51 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@google-cloud/firestore@npm:^5.0.1":
-  version: 5.0.1
-  resolution: "@google-cloud/firestore@npm:5.0.1"
+"@google-cloud/bigquery@npm:^5.9.3":
+  version: 5.9.3
+  resolution: "@google-cloud/bigquery@npm:5.9.3"
+  dependencies:
+    "@google-cloud/common": ^3.1.0
+    "@google-cloud/paginator": ^3.0.0
+    "@google-cloud/promisify": ^2.0.0
+    arrify: ^2.0.1
+    big.js: ^6.0.0
+    duplexify: ^4.0.0
+    extend: ^3.0.2
+    is: ^3.3.0
+    p-event: ^4.1.0
+    stream-events: ^1.0.5
+    uuid: ^8.0.0
+  checksum: ff8bbe7878d6ecfcbf181aeeb6b33ab8c1a64aa89b7f374192b2e9b59e82e396ca23856f41549696f3335e3cf8143aa53c0b15d860f32091a6095fc5d822fc0a
+  languageName: node
+  linkType: hard
+
+"@google-cloud/common@npm:^3.1.0":
+  version: 3.8.1
+  resolution: "@google-cloud/common@npm:3.8.1"
+  dependencies:
+    "@google-cloud/projectify": ^2.0.0
+    "@google-cloud/promisify": ^2.0.0
+    arrify: ^2.0.1
+    duplexify: ^4.1.1
+    ent: ^2.2.0
+    extend: ^3.0.2
+    google-auth-library: ^7.9.2
+    retry-request: ^4.2.2
+    teeny-request: ^7.0.0
+  checksum: b1b384fe066572c3b221013837e9423f62c501e04dc820012642449b6faf0f95b3b5aea9521a9fc8ff1ab4a4ee28cf170924d981205077fdc076c4be86312d86
+  languageName: node
+  linkType: hard
+
+"@google-cloud/firestore@npm:^5.0.2":
+  version: 5.0.2
+  resolution: "@google-cloud/firestore@npm:5.0.2"
   dependencies:
     fast-deep-equal: ^3.1.1
     functional-red-black-tree: ^1.0.1
     google-gax: ^2.24.1
     protobufjs: ^6.8.6
-  checksum: 2a2c6272bc01bc9c850e091d2fe44b777b32162980fdb085574ca86d2c7a2014b7b5b96dc9a5593048706ea3815f9662e344a0992e577197974fe7cb1e79f6b4
+  checksum: 4b669f54a7da2287d0559435586bcf9792a3e240d1b75868a6557c4c1a7265aae335ab24d0d1458ed32320ee258c107802f9650ed24f86b0d03ff49f202ee530
```

Comment:
There was a merge conflict with train 223 on the `firestore` version, so I made it match the version (5.0.2) and checksum in `main`.